fs/config/configflags/configflags.go

```diff
@@ -124,6 +124,7 @@ func AddFlags(ci *fs.ConfigInfo, flagSet *pflag.FlagSet) {
 	flags.StringArrayVarP(flagSet, &downloadHeaders, "header-download", "", nil, "Set HTTP header for download transactions")
 	flags.StringArrayVarP(flagSet, &headers, "header", "", nil, "Set HTTP header for all transactions")
 	flags.BoolVarP(flagSet, &ci.RefreshTimes, "refresh-times", "", ci.RefreshTimes, "Refresh the modtime of remote files.")
+	flags.BoolVarP(flagSet, &ci.HideConsole, "no-console", "", ci.HideConsole, "Hide console window. Supported on Windows only.")
```

Comment:
I try (I don't always succeed!) to make the internal names `HideConsole` agree with the flag names `--no-console` - so let's pick one or the other! Note that the internal names show in the `rc`.

Reply:
Makes sense. Function is called `HideConsole`, and flag is called `no-console`, so internal name matched the function name instead of the flag name. Switched that. Could have renamed the function also to `NoConsole` but think it is more correct to call it `HideConsole` since that is what it *does*.